apps/editing-toolkit/editing-toolkit-plugin/whats-new/src/whats-new.js

```diff
@@ -0,0 +1,247 @@
+/*** THIS MUST BE THE FIRST THING EVALUATED IN THIS SCRIPT *****/
+import './public-path';
+
+/**
+ * External dependencies
+ */
+import { registerPlugin } from '@wordpress/plugins';
+import { Fill, Guide, GuidePage, MenuItem } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useEffect, createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+const paymentsImage = 'https://s0.wp.com/i/whats-new/payments.png';
+const whatsappImage = 'https://s0.wp.com/i/whats-new/whatsapp.png';
+const blockPatternsImage = 'https://s0.wp.com/i/whats-new/block-patterns.png';
+const dragDropImage = 'https://s0.wp.com/i/whats-new/drag-drop.png';
+const singlePageSiteImage = 'https://s0.wp.com/i/whats-new/single-page-website.png';
+
+function WhatsNewMenuItem() {
+	const { toggleWhatsNew } = useDispatch( 'whats-new' );
+	const isActive = useSelect( ( select ) => select( 'whats-new' ).isWhatsNewActive() );
+	const whatsNewPages = getWhatsNewPages();
+
+	// Record Tracks event if user opens What's New
+	useEffect( () => {
+		if ( isActive ) {
+			recordTracksEvent( 'block_editor_whats_new_open' );
+		}
+	}, [ isActive ] );
+
+	return (
+		<>
+			<Fill name="ToolsMoreMenuGroup">
+				<MenuItem onClick={ () => toggleWhatsNew() }>
+					{ __( "What's New", 'full-site-editing' ) }
+				</MenuItem>
+			</Fill>
+			{ isActive && (
+				<Guide
+					className="whats-new"
+					contentLabel={ __( "What's New at WordPress.com", 'full-site-editing' ) }
+					finishButtonText={ __( 'Close', 'full-site-editing' ) }
+					onFinish={ () => toggleWhatsNew() }
```

Comment:
```suggestion
					onFinish={ toggleWhatsNew }
```

Nit: you can just reference the function as a variable directly :)